include/exadg/convection_diffusion/spatial_discretization/operator.cpp

```diff
@@ -311,8 +328,13 @@ Operator<dim, Number>::setup_operators()
 
 template<int dim, typename Number>
 void
-Operator<dim, Number>::setup()
+Operator<dim, Number>::setup(bool const verbosity)
 {
+  if(verbosity)
+  {
+    pcout << std::endl << "Setup convection-diffusion operator ..." << std::endl;
+  }
```

Comment:
I would like to avoid introducing boolean parameters for these public member functions.

Let me make another suggestion that could be more elegant in terms of the interfaces of this class:

```C++
public:
void setup()
{
  // write output

  // call private member function
  do_setup();

 // write output
}

private:
void do_setup(){...}
```

And then call `do_setup()` in `setup_after_coarsening_and_refinement()`. This should allow to avoid the parameters and if-statements.